packages/language-server/test/snapshots/xml-view-diagnostics/non-stable-id/output-lsp-response.json

```diff
@@ -7,7 +7,7 @@
     "severity": 1,
     "source": "UI5 Language Assistant",
     "message": "The class \"Button\" must declare a non-empty ID attribute when flexEnabled is \"true\"",
```

Comment:
"The \"Button\" class can't have an empty ID attribute when flexEnabled is \"true\".